src/index.html

```diff
@@ -22,6 +22,408 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <div class="header">
+      <a
+        href="#"
+        class="moyologo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="moyo_logo_image"
```

Comment:
For logos, use the format: "[Organization name] logo." 

![image](https://github.com/mate-academy/layout_catalog/assets/54860641/e10c2e09-8598-4d7a-b71e-df48cb9e28e7)


```suggestion
          alt="Moyo logo"
```